.github/renovate.json5

```diff
@@ -53,6 +53,19 @@
   // Don't leave dep. update. PRs "hanging", assign them to people.
   "assignees": ["rhatdan", "vrothberg", "Luap99"],
 
+  "customManagers": [
```

Comment:
We have a shared bit of configuration for this in https://github.com/containers/automation/blob/main/renovate/defaults.json5 . That can’t be directly used here as is, but I’d prefer extending that configuration to maintaining a separate local copy.

Reply:
I am aware of the shared configs. That said I am not aware of another repo that uses that syntax so I much rather have the bits here where they end up being used.
If someone fucks up the automation config they break all repos, if this one here isn't working I just break one repo and can fix it without much harm.